signcrypt_seal.go

```diff
@@ -69,6 +69,7 @@ func (sss *signcryptSealStream) signcryptBytes(b []byte) error {
 
 	nonce := nonceForChunkSigncryption(sss.numBlocks)
 
+	// TODO: Encode payload chunk.
```

Comment:
Ah, I misread it. I get it now thanks!

I guess if we want to MAC the isFinal flag, we'd just add a byte to be sha512ed (same with the encryption case). I was thinking we might want to msgpack-encode it as an array, but that seems like overkill. WDYT?